config/config.go

```diff
@@ -155,6 +155,7 @@ var (
 		HonorLabels:             false,
 		HonorTimestamps:         true,
 		HTTPClientConfig:        config.DefaultHTTPClientConfig,
+		KeepDroppedTargets:      100,
```

Comment:
This reads like a boolean option, but I failed to quickly come up with a nice-sounding option that sounds more like a numeric limit, so I guess it's fine unless someone has a better idea :)